.redhat-ci.yml

```diff
@@ -0,0 +1,15 @@
+host:
+    distro: fedora/24/cloud
```

Comment:
I'll update to use Atomic since it's supported today by redhat-ci. They should be able to run via docker as well, but I'll wait for @jlebon's additions. 

Once we port and enable our BDD tests we will need VM's to test different OS versions/configurations but the unit tests and container tests can continue in docker and Atomic.